deploy/assisted-service-configmap.yaml

```diff
@@ -15,4 +15,5 @@ data:
   JWKS_URL: REPLACE_JWKS_URL # example https://example.com/.well-known/jwks.json
   OCM_BASE_URL: REPLACE_OCM_BASE_URL
   OCM_DEBUG: "false"
-  SKIP_CERT_VERIFICATION: "true"
\ No newline at end of file
+  SKIP_CERT_VERIFICATION: "true"
+  LOG_LEVEL: "info"
```

Comment:
No need to set it here if we already have a default in the code
And if you set it then lets set it in the deployment file. otherwise we will have it defined in the configmap and in the deployment file.
If you set it in the configmap and then change it then you will need to reboot the service, setting it in the deployment will reboot the service for you.

Reply:
Moved to deployment